content/en/docs/concepts/containers/runtime-class.md

```diff
@@ -42,14 +42,14 @@ kubelets.
 
 #### 2. Install the RuntimeClass CRD
 
-The RuntimeClass [CustomResourceDefinition][/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/] (CRD) can be found in the addons directory of the
+The RuntimeClass [CustomResourceDefinition][] (CRD) can be found in the addons directory of the
 Kubernetes git repo:
 
```

Comment:
- Do you want to create a link from some of the text so that the link is not explicitly listed, such as:
The RuntimeClass [CustomResourceDefinition][] (CRD), [runtimeclass_crd.yaml](https://github.com/kubernetes/kubernetes/tree/release-1.12/cluster/addons/runtimeclass/runtimeclass_crd.yaml), can be found in the addons directory of the Kubernetes github repo.

Reply:
Sorry, I don't understand your question. Can you please clarify?

Reply:
I am suggesting to remove line 48, and instead hide the raw link.
The RuntimeClass [CustomResourceDefinition][] (CRD), [runtimeclass_crd.yaml](https://github.com/kubernetes/kubernetes/tree/release-1.12/cluster/addons/runtimeclass/runtimeclass_crd.yaml), can be found in the addons directory ...

Reply:
How this?